tests/LexerTest.php

```diff
@@ -353,6 +353,26 @@ public function testUnterminatedBlock()
         $lexer->tokenize(new Source($template, 'index'));
     }
 
+    public function testCommentValues()
+    {
+        $template = '{# comment #}some text{#another one#}';
+        $lexer = new Lexer(new Environment($this->createMock(LoaderInterface::class)));
```

Comment:
```suggestion
        $lexer = new Lexer(new Environment(new ArrayLoader()));
```

As this is how it's done in 4.x.